src/App.js

```diff
@@ -1,19 +1,45 @@
 import React from 'react';
 import './App.css';
-import chatMessages from './data/messages.json';
+import chatMessages from './data/messages.json'
+// import ChatEntry from './components/ChatEntry'
+import ChatLog from './components/ChatLog'
+import {useState} from 'react';
 
 const App = () => {
+  const [chatEntry, setChatEntry] = useState(chatMessages)
+
+  const updateChatData = updatedChat => {
+    const newChats = chatEntry.map(chat => {
+      if (chat.id === updatedChat.id) {
+        return updatedChat;
+      } else {
+        return chat;
+      }
+    });
+
+    setChatEntry(newChats);
+  };
+
+  const increaseLikes = (chatEntry) => {
+    let likeCount = 0
+    for (let chat of chatEntry) {
+      if (chat.liked === true) {
+        likeCount ++
+      }
```

Comment:
We could use JS's logical and operator here instead of an `if` conditional and refactor these three lines into:
```
chat.liked && likeCount++
```